include/effects/matrix/PatternSwirl.h

```diff
@@ -73,27 +73,25 @@ class PatternSwirl : public LEDStripEffect
 
     void drawAt(int i, int j, CRGB color)
     {
-        auto graphics = _GFX[0];
+        auto graphics = g();
 
         graphics->leds[graphics->xy(i, j - 1)] += color;
         graphics->leds[graphics->xy(i, j + 1)] += color;
         graphics->leds[graphics->xy(i - 1, j)] += color;
         graphics->leds[graphics->xy(i + 1, j)] += color;
         color.maximizeBrightness();
-        graphics->leds[graphics->xy(i, j)] += color;
+        graphics->leds[XY(i, j)] += color;
     }
 
     void Draw() override
     {
-        auto graphics = _GFX[0];
-
```

Comment:
I'd change this to set `graphics` to `g()`, and then keep using `graphics` further down in this function. Contrary to `drawAt()`, where I'd indeed remove `graphics` and call `g()` directly on line 83.